cmd/commands/runtime_install.go

```diff
@@ -106,17 +106,17 @@ type (
 		ExternalIngressAnnotation      map[string]string
 		EnableGitProviders             bool
 		AccessMode                     platmodel.AccessMode
-		InstallFeatures                []runtime.InstallFeature
 		TunnelRegisterHost             string
 		TunnelDomain                   string
 		TunnelSubdomain                string
 		SkipIngress                    bool
 
-		versionStr    string
-		kubeContext   string
-		kubeconfig    string
-		gitProvider   cfgit.Provider
-		useGatewayAPI bool
+		versionStr      string
+		kubeContext     string
+		kubeconfig      string
+		gitProvider     cfgit.Provider
+		useGatewayAPI   bool
+		installFeatures []runtime.InstallFeature
```

Comment:
just moved to be a private field, no special reason (it is maintained internally)